main.go

```diff
@@ -108,13 +108,14 @@ var DepMapCommand = cli.Command{
 	Name:  "dep-map",
 	Usage: "prints out a json dep map for usage by 'import --map'",
 	Action: func(c *cli.Context) error {
-		pkg, err := LoadPackageFile(gx.PkgFileName)
+		var pkg gx.Package
```

Comment:
any particular reason to do this? The base `gx.Package` type doesnt have the platform specific code in it

Reply:
Hm. I believe I thought I needed it to call `ForEachDep` but that's clearly not the case.

Reply:
Ah. Yes, it is because I wanted to use `ForEachDep`.

1. I don't actually use the additional fields.
2. I need `buildMap` to take a `gx.Package` because that's what `ForEachDep` spits out.

Reply:
> If you make buildmap take a gx.PackageBase, then the gx-go Package type can be passed into it just fine.

Yeah, I'd just rather be consistent. Note: I *do* need a gx field `dvcsImport` but I extract that with `GxDvcsImport`.